app/browser/webtorrent.js

```diff
@@ -145,5 +145,7 @@ function getHeader (headers, headerName) {
 
 module.exports = {
   init,
+  isTorrentFile,
+  isMagnetURL,
```

Comment:
this approach looks good, but FWIW: @feross is it intentional that these methods check `details.url` instead of `details.firstPartyUrl`?